viewer/packages/api/src/public/RevealManager.ts

```diff
@@ -25,7 +25,8 @@ import { MetricsLogger } from '@reveal/metrics';
 import { assertNever, EventTrigger } from '@reveal/utilities';
 import { CameraManager } from '@reveal/camera-manager';
 
-import { ModelIdentifier } from '@reveal/data-providers';
+import { ClassicModelIdentifierType, InternalDataSourceType, LocalModelIdentifierType } from '@reveal/data-providers';
+import { createModelIdentifier } from '@reveal/data-providers/src/ModelIdentifier';
```

Comment:
Import path fix!
Would need to add `createModelIdentifier` into `data-provider` `index.ts` file